src/App.php

```diff
@@ -265,13 +265,21 @@ protected function configureRoutes( Slim $slim ) {
 				$middleware['inject-user'], $middleware['set-environment'],
 			function () use ( $slim, $middleware, $routeConditions ) {
 				$slim->get( '', function() use ( $slim ) {
+					// See if we have a cookie indicating last version used
+					if ( isset( $_COOKIE['copypatrolLang'] ) ) {
+						$slim->redirectTo( 'home', [ 'wikiLang' => $_COOKIE['copypatrolLang'] ] );
+					} else {
+						// If no cookie, redirect to English version
+						$slim->redirectTo( 'home', [ 'wikiLang' => 'en' ] );
+					}
 					// Redirect root-URL requests to English.
 					$currentLang = $slim->i18nContext->getCurrentLanguage();
 					$slim->redirectTo( 'home', [ 'wikiLang' => $currentLang ] );
```

Comment:
Yep, I'm now realizing that as well. So I believe getting rid of the `else` block on lines 271-274 will do what we want. What do you think @Niharika29 ?